### DISCUSSION TITLE
Network 10.10.x.x (for 65535 client)

### DISCUSSION THREAD

**Discussion by wawanUnic** [wg-easy/wg-easy#1022] [Q&A]
Hello.
Tell me what to do if I want to create a network 192.168.х.х. for 65535 clients?
Will WG-easy be able to work like this?
I will need to change /etc/systemd/system/wg-easy.service line?
_Environment="WG_DEFAULT_ADDRESS=10.10.x.x" #Clients IP address range_
Thank you.

**Comment by TheRealC0unt** [comment]
Technically you simply set up the ip and range in CIDR notation. But you should ask yourself whether WG is the proper decision for more than 254 clients so far.

**Comment by pheiduck** [comment]
We have a feature like this in WIP but it is in early state as it had breaking some functionality in the past…
